build-system/tasks/karma.conf.js

```diff
@@ -227,9 +227,6 @@ module.exports = {
       // Longer timeout on Travis; fail quickly at local.
       timeout: process.env.TRAVIS ? 10000 : 2000,
     },
-    // TODO(rsimha, #14406): Remove this after all tests are fixed.
-    failOnConsoleError: !process.env.TRAVIS && !process.env.LOCAL_PR_CHECK,
-    // TODO(rsimha, #14432): Set to false after all tests are fixed.
```

Comment:
This TODO is about the next line, I assume. Are you removing it on purpose as well?

Reply:
Yes. We want to continue capturing console errors.